app/scripts/views/base.js

```diff
@@ -58,11 +67,20 @@ function (Cocktail, _, Backbone, Raven, $, p, AuthErrors,
     this.logError(err);
     var translated = this.translateError(err);
 
+    var $error = this.$('.error');
     if (translated) {
-      this.$('.error')[displayStrategy](translated);
+      $error[displayStrategy](translated);
     }
 
-    this.$('.error').slideDown(EPHEMERAL_MESSAGE_ANIMATION_MS);
+    // the 'data-was-shown' attribute value is added so the functional tests
+    // can find out if the error message was successfully shown, even
+    // if the element is then hidden. In the functional tests,
+    // testErrorWasShown removes the attribute so multiple checks for the
+    // element can take place in the same test.
+    $error
+      .slideDown(EPHEMERAL_MESSAGE_ANIMATION_MS)
+      .attr('data-was-shown', 'true');
```

Comment:
Maybe instead of `data-was-shown` here and `data-ready` in `legal_copy.js`, use `data-visible` in both to stay consistent.